maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -212,7 +209,7 @@ class IOSDriver(
 
         val contentDescriptorResult = iosDevice.contentDescriptor(
             resolvedAppId ?: throw IllegalStateException("Failed to get view hierarchy, app id was not resolvedGetRunningAppRequest.kt")
-        )
+        ) { errorMessage -> logger.info(errorMessage) }
```

Comment:
Should we instead add an `error` function and call `logger.error` here?

Reply:
Unfortunately, there is no logger error for the `DebugLogStore` class. I'll start a thread on this and let's solve that separately.